tests/mocks.py

```
@@ -0,0 +1,495 @@
+        """
+        super().__init__(message)
+        self.message: str = message
+
+
+class MockSyncError(Exception):
+    def __init__(
+        self,
+        message: str = "Sync error",
+        status_code: str | None = None,
+        retry_after_ms: int | None = None,
+        soft_logout: bool = False,
```

Ruff (0.15.12)

[warning] Boolean-typed positional argument in function definition

(FBT001)